host_core/lib/host_core/policy/manager.ex

```diff
@@ -71,7 +72,7 @@ defmodule HostCore.Policy.Manager do
           publicKey: host_config.host_key,
           issuer: host_config.cluster_key,
           latticeId: host_config.lattice_prefix,
-          labels: host_config.labels,
+          labels: labels,
```

Comment:
`host_config` does not contain a `:labels` key, causing this to silently fail every time the function is called